mappings/net/minecraft/world/GameRules.mapping

```diff
@@ -38,8 +38,9 @@ CLASS net/minecraft/class_1928 net/minecraft/world/GameRules
 	FIELD field_9197 RULE_TYPES Ljava/util/Map;
 	METHOD <init> (Ljava/util/Map;)V
 		ARG 1 rules
-	METHOD method_20744 forEachType (Lnet/minecraft/class_1928$class_4311;)V
-		ARG 0 action
+	METHOD method_20744 visit (Lnet/minecraft/class_1928$class_4311;)V
```

Comment:
I would name this `accept` or `acceptAll`, elements typically use accept(visitor) to call visitor.visitXY(element).